### ISSUE TITLE
cropping size

### ISSUE THREAD

**Issue by guozixunnicolas** [ly-atdawn/Unet-Tensorflow#2]
Hi,

I am a bit confused about the crop coefficient used in your code.

In crop 1, original size is 568*568, target size is 392*392, so (568-392)/2=88, may I know why you use crop size 90 in your code? (the rest of crop size from my calculation is 40,16,4 and the coefficient you used is 41,(16,17),and4) 

May I know the reason behind?

Best regards,

ZX

**Comment by lyatdawn** [comment]
Sorry, which codes?

**Comment by guozixunnicolas** [comment]
for example: crop1 = tf.keras.layers.Cropping2D(cropping=((90, 90), (90, 90)))(conv1_2)
in model.py. Could you please tell me why you used 90?

Best regards,


Zx

**Comment by lyatdawn** [comment]
90 is a self defined param, use Cropping2D(), the output mask is smaller than input. Of course, you can delete this Cropping2D layer.

**Comment by guozixunnicolas** [comment]
Thank you!

**Comment by lishi0927** [comment]
I have the same question about crop2D function.
The tensor from crop2D function will be concatenated with the upsampling function.
For example,
crop4 = tf.keras.layers.Cropping2D(cropping=((4, 4), (4, 4)))(drop4)
up6 = tf.layers.conv2d(up6_1, 512, 2, padding="SAME", activation=tf.nn.relu,
            kernel_initializer = tf.contrib.layers.xavier_initializer())
merge6 = tf.concat([crop4, up6], axis=3) # concat channel

The dimension of these two tensors are the same dimension in the second and third axis? what the dimension of the crop4?(batch, ?, ?, channel)
